scheduler/java/com/netflix/fenzo/SimpleAssignmentResult.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) Two Sigma Open Source, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.fenzo;
+
+import java.util.List;
+import java.util.Map;
+
+
+public class SimpleAssignmentResult extends TaskAssignmentResult {
```

Comment:
It looks like we only use this in tests. It seems simpler to just `extend` TaskAssignmentResult in Clojure than introduce Java to the project.